### ISSUE TITLE
Horizontal layout not expanding width before drop

### ISSUE THREAD

**Issue by danielcarlstrom** [kutlugsahin/ngx-smooth-dnd#16]
Hi,

First:
### Thanks for a great tool!

Now, I'm trying to build a horizontal layout with groups where you can drop in between groups.
I'm not sure on how to make the group container expand when being hovered with an item.

If I use vertical, the height is altered, but not when it's horizontal.

Please see this simple, not fully working, example. It shows the flaw.
https://stackblitz.com/edit/angular-mqyve9

It's probably something I've missed but would be happy for some guidance.

Thanks,

**Comment by danielcarlstrom** [comment]
Okay I think I found it, the placeholder element that is inserted into the container when hovering with a droppable item needs to be inline-block, then it works, so I added this to css:

```
::ng-deep.smooth-dnd-stretcher-element.horizontal {
  display: inline-block;
}
```

**Comment by kutlugsahin** [comment]
Hey @danielcarlstrom. Thanks for submitting the issue. I applied your solution. Thanks to you it is fixed at version 0.2.5 :)
Cheers!!
